### ISSUE TITLE
external plugins: handle permission denied properly

### ISSUE THREAD

**Issue by ktsaou** [firehol/netdata#407]
netdata should detect it does not have permissions to execute external plugins and disable them.
Now, netdata retries forever to execute them (every 10 seconds).

```
16-05-13 23:36:54: INFO (0137@plugins_d.:pluginsd_worker): netdata: PLUGINSD: '/usr/libexec/netdata/plugins.d/apps.plugin' running on pid 22582
16-05-13 23:36:54: INFO (0136@popen.c   :mypopen        ): netdata: executing command: 'exec /usr/libexec/netdata/plugins.d/apps.plugin 1 ' on pid 22582.
sh: 1: exec: /usr/libexec/netdata/plugins.d/apps.plugin: Permission denied
16-05-13 23:36:54: INFO (0390@plugins_d.:pluginsd_worker): netdata: PLUGINSD: '/usr/libexec/netdata/plugins.d/apps.plugin' on pid 22582 stopped.
16-05-13 23:36:54: ERROR (0151@popen.c   :mypclose       ): netdata: pid 22582 exited with code 126.
16-05-13 23:36:54: ERROR (0410@plugins_d.:pluginsd_worker): netdata: PLUGINSD: '/usr/libexec/netdata/plugins.d/apps.plugin' (pid 22582) does not generate useful output. Waiting a bit before starting it again.
```

I guess exit code 126 is the key here.

**Comment by CharlesRaymond1** [comment]
Would it be a good alternative to retry with a configurable amount of time or even hook up with inotify?

**Comment by ktsaou** [comment]
Nice idea!
@simonnagl FYI, another job for the polling thread...

**Comment by simonnagl** [comment]
Sorry, I don't get your idea. For what do you like to poll here?

**Comment by ktsaou** [comment]
> Sorry, I don't get your idea. For what do you like to poll here?

inotify for new / updated plugins

**Comment by ktsaou** [comment]
Merged a fix with #525.
The inotify will be implemented later.
